pkg/webui/console/views/device-general-settings/identity-server-form/index.js

```diff
@@ -0,0 +1,288 @@
+// Copyright © 2019 The Things Network Foundation, The Things Industries B.V.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import React from 'react'
+import { defineMessages } from 'react-intl'
+
+import SubmitButton from '../../../../components/submit-button'
+import SubmitBar from '../../../../components/submit-bar'
+import Input from '../../../../components/input'
+import Form from '../../../../components/form'
+import Checkbox from '../../../../components/checkbox'
+import ModalButton from '../../../../components/button/modal-button'
+
+import diff from '../../../../lib/diff'
+import m from '../../../components/device-data-form/messages'
+import PropTypes from '../../../../lib/prop-types'
+import sharedMessages from '../../../../lib/shared-messages'
+import { parseLorawanMacVersion, hasExternalJs, isDeviceOTAA } from '../utils'
+import validationSchema from './validation-schema'
+
+const messages = defineMessages({
+  deleteDevice: 'Delete End Device',
+  deleteWarning:
+    'Are you sure you want to delete "{deviceId}"? Deleting an end device cannot be undone!',
```

Comment:
```suggestion
    'Are you sure you want to delete "{deviceId}"? This action cannot be undone and it will not be possible to reuse the end device ID!',
```